mlflow/utils/_spark_utils.py

```diff
@@ -36,6 +37,44 @@ def _prepare_subprocess_environ_for_creating_local_spark_session():
     os.environ.pop("PYSPARK_GATEWAY_SECRET", None)
 
 
+def _get_spark_scala_version_from_spark_session(spark):
+    version = spark._jvm.scala.util.Properties.versionNumberString().split(".")
+    return f"{version[0]}.{version[1]}"
+
+
+def _get_spark_scala_version():
+    from pyspark.sql import SparkSession
+
+    from mlflow.utils.databricks_utils import is_in_databricks_runtime
+
+    if is_in_databricks_runtime() and "SPARK_SCALA_VERSION" in os.environ:
+        return os.environ["SPARK_SCALA_VERSION"]
```

Comment:
```suggestion
    if is_in_databricks_runtime() and ver := os.environ.get("SPARK_SCALA_VERSION"):
        return ver
```

Reply:
Python doesn't support syntax of `if is_in_databricks_runtime() and ver := os.environ.get("SPARK_SCALA_VERSION"):`:

`SyntaxError: cannot use assignment expressions with operator`